src/ShopifyClient.php

```diff
@@ -529,7 +529,7 @@ public function wrapRequestData(CommandInterface $command): RequestInterface
      * @param  RequestException|null  $exception
      * @return bool
      */
-    public function retryDecider(int $retries, RequestInterface $request, ResponseInterface $response = null, RequestException $exception = null): bool
```

Comment:
Why did you change that? This is not a breaking change as far as I know but I don't think this is needed